index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Porfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav>
+      <p>Toyin</p>
+      <ul>
+        <li><a href="">About</a></li>
+        <li><a href="">Porfolio</a></li>
+        <li><a href="">Contact</a></li>
+      </ul>
+    </nav>
+    <header>
+      <div class="header-background"></div>
+      <div class="header-text">
+        <p>
+          Hi! My name is Toyin Ayorinde, I'm a front end developer from Lagos
+          Nigeria
+        </p>
+      </div>
+    </header>
+    <main>
+      <div class="about">
+        <div class="about-left">
+          <img src="./images/profile.JPG" alt="" />
```

Comment:
Don't you think adding a descriptive text as the value of the alt attribute would better?